modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java

```diff
@@ -85,6 +92,34 @@ public void s3TestOverBridgeNetwork() throws IOException {
             assertEquals("The object can be retrieved", "baz", content);
         }
 
+        @Test
+        public void s3TestOverBridgeNetworkV2() {
```

Comment:
1) I think test's name is a bit too specific
2) could you please reduce the test to the minimum, so that it only tests that SDK v2 works fine?